libs/output-mapping/src/Writer/TableWriter.php

```diff
@@ -103,11 +106,27 @@ public function uploadTables(
                 continue;
             }
 
+            if (!$strategy instanceof SqlWorkspaceTableStrategy) {
```

Comment:
Pro OM který nejde přes sql workspace validujeme timestamp

Reply:
Nevím, já to neprogramoval, ale řekl bych že jo?

Reply:
Mělo by https://github.com/keboola/platform-libraries/blob/e075ab44a8e751594a83d430db5dad38af5cdb2f/libs/output-mapping/src/Staging/StrategyFactory.php#L10